Kernel/Panic.cpp

```diff
@@ -8,6 +8,8 @@
 #include <Kernel/Arch/Processor.h>
```

Comment:
The changes in this file seems unrelated to the commit. You probably accidentally included this into this commit.

Reply:
Should be fixed now.